tools/deprecate_version/deprecate_version.py

```diff
@@ -116,6 +116,11 @@ def create_issues(access_token, runtime_and_pr):
         print('Creating issues...')
         for title, body, login in issues:
             try:
+                # for setec backports, we may not find a user, which will
+                # cause repo.create_issue to crash. Preempt this by hitting the
+                # exception block below.
+                if not login:
+                    raise github.GithubException("login required")
```

Comment:
this is a bit confusing as i think it might imply github api failed due to auth issues, that was certainly my initial reading of code - i guess you want to raise so it triggers the alternate path - perhaps move that out of except and set a marker